content/soarian/dstu2/common-errors.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Common Errors | Soarian DSTU 2 API
+---
+
+# CommonErrors
+
+### Common Errors
+The following type of errors may be returned with the API request.
+
+| HTTP Status code | Message |
+| --- | --- |
+|200|Success|
```

Comment:
@slvsridhar Will need your input on what that second column is? Is it the message or the reason it occurs?